CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+
+Install snapcraft:
+
+```bash
+sudo snap install snapcraft --classic
```

Comment:
```suggestion
sudo snap install snapcraft --devmode
```

Reply:
Ahh, sorry. Wrong place. It was supposed to go to the `sdcore-upf` install command.